core/playwright.config.ts

```diff
@@ -7,22 +7,27 @@ expect.extend(matchers);
 
 const projects = [
   {
-    name: 'chromium',
```

Comment:
Could you add a note to the internal Notion doc about the change? Might be good to also include a code snippet for adding desktop tests.

Reply:
Added to the "Writing Tests in Core" document under "Tablet Viewport Tests"